pkg/provider/provider.go

```diff
@@ -234,6 +235,7 @@ func buildTestEnvironment() { //nolint:funlen
 			logrus.Warnf("Pod %s has been deployed using a DeploymentConfig, please use Deployment or StatefulSet instead.", pod.String())
 		}
 	}
+	logrus.Infof("Completed the test environment build process in %d seconds", int(time.Since(start).Seconds()))
```

Comment:
Casting to `int` is weird. You can print `float` with `%.2f`. Or just print the duration as a string: `..."%s", time.Since(start)`.